modules/validations/duplicate_tags.js

```diff
@@ -0,0 +1,77 @@
+import { actionChangeTags } from '../actions/index';
+import { t } from '../util/locale';
+import { validationIssue, validationIssueFix } from '../core/validator';
+
+
+export function validationDuplicateTags() {
+    var type = 'duplicate_tags';
+    var tagsToCheck = [
+                        ['Name', ['name']],
+                        ['Address', ['addr:housenumber', 'addr:street']],
```

Comment:
Since this code only checks `addr:housenumber` and `addr:street`, it’ll incorrectly flag features that share `addr:housenumber` and `addr:street` but differ in `addr:unit`. This can happen for instance when there are multiple tenants in the same building, each mapped as a point.